src/pages/RegisterPage/RegisterPage.css

```diff
@@ -6,5 +6,21 @@
 }
 
 .form-footer {
-    font-weight: lighter;
+    font-style: italic;
+}
+
+h1 {
```

Comment:
It looks like these are the same css styles already declared in `index.css` so they are redundant here

Reply:
just switched branches to main and i don't see anything in the index.css there?